cg/apps/scout/scoutapi.py

```diff
@@ -184,67 +172,51 @@ def get_cases(
         for case_export in ReadStream.get_content_from_stream(
             file_format=FileFormat.JSON, stream=self.process.stdout
         ):
-            LOG.info("Validating case %s", case_export.get("_id"))
+            LOG.info(f"Validating case {case_export.get('_id')}")
             cases.append(ScoutExportCase(**case_export))
         return cases
 
     def get_solved_cases(self, days_ago: int) -> List[ScoutExportCase]:
         """
-        Get cases solved within chosen timespan
-
-        Args:
-            days_ago (int): Maximum days ago a case has been solved
-
-        Return:
-            cases (list): list of cases
+        Get cases solved within chosen time span.
         """
         return self.get_cases(status="solved", days_ago=days_ago)
 
     def upload_delivery_report(self, report_path: str, case_id: str, update: bool = False) -> None:
-        """Load a delivery report into a case in the database
-
+        """Load a delivery report into a case in the database.
         If the report already exists the function will exit.
         If the user want to load a report that is already in the database
-        'update' has to be 'True'
-
-        Args:
-            report_path (string):       Path to delivery report
-            case_id     (string):       Case identifier
-            update      (bool):         If an existing report should be replaced
-
-        Returns:
-            updated_case(dict)
-
+        'update' has to be 'True'.
         """
-        # This command can be run with `scout load delivery-report <CASE-ID> <REPORT-PATH>`
-        upload_command = ["load", "delivery-report", case_id, report_path]
+
+        upload_command: List[str] = ["load", "delivery-report", case_id, report_path]
 
         if update:
             upload_command.append("--update")
 
         try:
-            LOG.info("Uploading delivery report %s to case %s", report_path, case_id)
+            LOG.info(f"Uploading delivery report {report_path} to case {case_id}")
             self.process.run_command(upload_command)
         except CalledProcessError:
             LOG.warning("Something went wrong when uploading delivery report")
 
+    def upload_report(self, case_id: str, report_path: str, report_type: str) -> None:
+        """Load report into a case in the database. If the report already exists the function will exit."""
```

Comment:
What does it mean "the function will exit"? That the report is not re-uploaded nor error is raised?
If that is the case it might be good to, at least, log it